src/components/EmailNotifications/EmailNotificationsForm.js

```diff
@@ -272,24 +285,30 @@ function LegalTermsAndPolicy({ termsAccepted, setTermsAccepted }) {
             text-align: left;
           `}
         >
-          By continuing with your email, you agree to Aragon Court{' '}
+          By continuing with your email, you agree to Aragon Court's{' '}
           <Link href="https://anj.aragon.org/legal/terms-general.pdf">
-            legal terms{' '}
+            legal terms
           </Link>{' '}
           and{' '}
           <Link href="https://aragon.one/email-collection.md">
-            {' '}
-            email collection policy.
+            email collection policy
           </Link>
+          .
         </span>
       </div>
     </div>
   )
-}
+})
 
-const ActionButtons = styled(Button)`
-  width: ${({ compactMode }) =>
-    compactMode ? '100% ' : `calc((100% - ${2 * GU}px) /  2)`};
-`
+function ActionButtons({ compactMode, ...props }) {
```

Comment:
```suggestion
function ActionButton({ compactMode, ...props }) {
```

Nitpick, here and elsewhere, we should do this renaming. The plural suggests it's multiple buttons.